### ISSUE TITLE
Failed connect to termux using ssh key

### ISSUE THREAD

**Issue by wurenny** [Eugeny/tabby#5554]
# process
My Linux Mint terminal can connect to Termux's sshd service with ssh command. Remmina can also, but Tabby can't. I think this should be a problem of SSH protocol auto-negotiation. So far, Tabby Version 1.0.170 still not resolved

env:
* tabby version: 1.0.170
* termux version: 0.118.0

# reproduces
You can download and install termux on your phone from [termux.com](https://termux.com), then execute follow commands in termux:
```bash
pkg install openssh
ssh-keygen -m PEM -t rsa
cd .ssh
cat id_rsa.pub >> authorized_keys
chmod 600 authorized_keys
sshd
```
and execute follow commands in Linux Mint Terminal:
```bash
# firstly, copy termux's id_rsa text to Linux Mint .ssh directory as name: termux.rsa
ssh ${termux wifi ip} -i .ssh/termux.rsa -p 8002
```
This can log in successfully, but faild use Tabby to connect termux

**Comment by supersu097** [comment]
me too, still waiting for solution.

**Comment by wurenny** [comment]
> me too, still waiting for solution.

This problem has been fixed and I tested it's ok in version 1.0.187
